Dynamic-Landing-Page/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+  }
+  .wrapper {
+    width: 100vw;
+    height: 100vh;
+    background-repeat: round;
+    font-family: "Times New Roman", Times, serif;
+    color: rgba(5, 5, 5, 0.9);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    overflow: hidden;
+    text-align: center;
+  }
+  .time {
+    font-size: 10em;
+  }
+  .greeting {
+    font-size: 1.5em;
+    font-weight: normal;
+  }
+  .name {
+    font-size: 1.5em;
+    font-weight: normal;
+  }
+  .focus {
+    font-size: 1.5em;
+    font-weight: normal;
+  }
+  @media (max-width: 767px) {
+    .time {
+      font-size: 5em;
+    }
+    .greeting {
+      font-size: 0.75em;
+      font-weight: normal;
```

Comment:
no need to duplicate the font-weight for the mobile view here and below

Reply:
fixed